dockerfile/README.md

```diff
@@ -1,59 +1,94 @@
-# Dockerfile for WSO2 Enterprise Integrator #
-The Dockerfile defines the resources and instructions to build the Docker images with the WSO2 products and runtime configurations.
+# Dockerfiles for WSO2 Enterprise Integrator #
+This section defines dockerfiles and step-by-step instructions to build docker images for multiple run-times <br>
```

Comment:
Might be better to write the word runtime without a dash. WDYT?

Reply:
Agreed. Will update. Thanks.